src/devices/video/315_5124.cpp

```diff
@@ -391,6 +391,26 @@ void sega315_5124_device::set_frame_timing()
 
 
 u8 sega315_5124_device::vcount_read()
+{
+	u8 vc = vcount();
+
+	/*
+	osd_printf_verbose(
+		"fr %lld vpos %3d hpos %3d hc 0x%02X %s: vcount read value 0x%02X\n"
+		, screen().frame_number()
+		, screen().vpos()
+		, screen_hpos()
+		, hcount()
+		, machine().describe_context().c_str()
+		, vc
+	);
+	*/
```

Comment:
You should really use `logmacro.h` for configurable logging.  Commented code like this has a tendency to rot.  Also, the `logerror` and `osd_printf_*` functions don’t need size qualifiers (`%d` works with any size integer), and they can use `std::string` directly without needing to use `.c_str()`.

Oh, and while I’m commenting on this, we don’t tend to use comma at the beginning of the line when breaking up function arguments across lines, we’d usually do it like:
```
	func(
			arg,
			arg,
			arg);
```